parlai/core/torch_ranker_agent.py

```diff
@@ -387,14 +387,11 @@ def _set_label_cands_vec(self, *args, **kwargs):
         Useful to override to change vectorization behavior.
         """
         obs = args[0]
-        cands_key = (
-            'candidates'
-            if 'labels' in obs
-            else 'eval_candidates'
-            if 'eval_labels' in obs
-            else None
-        )
-        if cands_key is not None and self.opt[cands_key] not in [
+        if 'labels' in obs:
+            cands_key = 'candidates'
+        else:
+            cands_key = 'eval_candidates'
```

Comment:
this changes the semantics of above ever so slightly.

Reply:
it is meant to

Reply:
just checking